db/crud.go

```diff
@@ -281,10 +281,10 @@ func (db *Database) getRev(docid, revid string, maxHistory int, historyFrom []st
 	if revIDGiven {
 		// Get a specific revision body and history from the revision cache
 		// (which will load them if necessary, by calling revCacheLoader, above)
-		revision, err = db.revisionCache.GetWithCopy(docid, revid, copyType)
+		revision, err = db.revisionCache.Get(docid, revid, copyType)
 	} else {
 		// No rev ID given, so load active revision
-		revision, err = db.revisionCache.GetActive(docid, db.DatabaseContext)
+		revision, err = db.revisionCache.GetActive(docid, db.DatabaseContext, BodyShallowCopy)
```

Comment:
Although using BodyShallowCopy here preserves the previous handling, it feels more correct to pass in copyType here now that it's been added to the GetActive interface.

Reply:
Makes sense, thanks.